Names-values.Rmd

```diff
@@ -465,7 +465,7 @@ References also make it challenging to think about the size of individual object
 obj_size(x, y)
 ```
 
-Finally, R 3.5.0 and later have a feature that might lead to surprises: ALTREP, short for __alternative representation__. This allows R to represent certain types of vectors very compactly. The place you are most like to see this is with `:` because instead of storing every single number in the sequence, R just stores the first and last number. This means that every sequence, no matter how large, is the same size:
+Finally, R 3.5.0 and later versions have a feature that might lead to surprises: ALTREP, short for __alternative representation__. This allows R to represent certain types of vectors very compactly. The place you are most like to see this is with `:` because instead of storing every single number in the sequence, R just stores the first and last number. This means that every sequence, no matter how large, is the same size:
```

Comment:
"R 3.5.0 and later versions" doesn't sound right to me either, but I can't think of a wording that sounds better.

Reply:
How about this option? "Finally, from R 3.5.0 implemented a feature..." or "Finally, R 3.5.0 and forward have a feature..."